torchvision/datasets/cub2011.py

```diff
@@ -0,0 +1,31 @@
+import os
+import torch
+import torch.utils.data as data
+import torchvision
+from torchvision.datasets import ImageFolder
+from torchvision.datasets import CIFAR10
+
+class Cub2011(ImageFolder, CIFAR10):
+    base_folder = 'CUB_200_2011/images'
+    url = 'http://www.vision.caltech.edu/visipedia-data/CUB-200-2011/CUB_200_2011.tgz'
+    filename = 'CUB_200_2011.tgz'
+    tgz_md5 = '97eceeb196236b17998738112f37df78'
+    
+    train_list = [
+        ['001.Black_footed_Albatross/Black_Footed_Albatross_0001_796111.jpg', '4c84da568f89519f84640c54b7fba7c2'],
+        ['002.Laysan_Albatross/Laysan_Albatross_0001_545.jpg', 'e7db63424d0e384dba02aacaf298cdc0'],
+    ]
+    test_list = [
```

Comment:
If we plan to levearge `CIFAR10` as a super class (see comment above), then perhaps these can be auto generated from the `train_test_split.txt` in the tarfile.